jersea/src/index.js

```diff
@@ -13,7 +13,9 @@ ReactDOM.render(
   <React.StrictMode>
     <BrowserRouter>
       <DataProvider>
-        <App />
+        <AuthProvider>
+          <App />
+        </AuthProvider>
```

Comment:
Better to put Auth Provider as a upper layer of DataProvider

Reply:
ok will update